uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/absence-tab-custom-work-pattern-modal.component.js

```diff
@@ -107,12 +107,12 @@ define([
     }
 
     /**
-     * Loads the Job Contract Revision Change Reasons and indexes by `value`
+     * Loads the Leave and Absences Work Pattern Change Reasons and indexes by `value`
      *
      * @return {Promise}
```

Comment:
Please add info on what the Promise resolves with:

```js
* @return {Promise} resolves with {Array}
```

I ask this because I am not sure in the comment to the function above, I am not not sure what this func resolves: array or indexed object.

Reply:
It resolves with Array of object. Will update as suggested. Thanks